src/logic/project/js/js.ts

```diff
@@ -4,7 +4,7 @@ import { OutputStream } from '../../../common/output-stream';
 import { SupportedLanguages } from '../../application-code';
 
 export async function prepareJsProject(
-  sdkVerion = '3.0.0-alpha.12',
+  sdkVerion = '3.0.0-beta.0',
```

Comment:
Isn't it better to pin `@beta` tag instead of the exact version slug?

Reply:
@kysely you mean this:
 "@superfaceai/one-sdk": "beta",
?

Reply:
Yes

Reply:
Because the actual beta versions will change but the tag keeps the “latest beta” https://www.npmjs.com/package/@superfaceai/one-sdk?activeTab=versions

Reply:
Cool, I will fix the tests and it should be ready to go.